### PR TITLE
Extract model and rpc crates from runtime

### PR DESCRIPTION
**Pull request by antiguru**
Signed-off-by: Moritz Hoffmann <moritz.hoffmann@inf.ethz.ch>

### CODE REVIEW

**Comment by gandro** [comment]
Nice! Any indication whether it helps with compilation times?

File: src/strymon_runtime/src/lib.rs [hunk 1]
```diff
@@ -23,16 +23,13 @@ extern crate timely_communication;
 extern crate rand;
 
 extern crate serde;
-#[macro_use]
-extern crate serde_derive;
-
-extern crate abomonation;
-#[macro_use]
-extern crate abomonation_derive;
```
Comment (gandro) [inline src/strymon_runtime/src/lib.rs:31]: Would you mind removing these crates from the Cargo.toml as well?

**Comment by antiguru** [comment]
It doesn't affect compilation time, I only saw a negligible difference.

File: src/strymon_runtime/src/lib.rs (hunk 1 above)
Comment (antiguru) [reply src/strymon_runtime/src/lib.rs:31]: Thanks, changed in cb58534.

**Review by gandro — APPROVED** [review]

**Comment by gandro** [comment]
Thanks!

**Pull request merged**